src/main/java/dynamok/sink/DynamoDbSinkTask.java

```diff
@@ -96,16 +118,21 @@ public void put(Collection<SinkRecord> records) {
         try {
             if (records.size() == 1 || config.batchSize == 1) {
                 for (final SinkRecord record : records) {
-                    client.putItem(tableName(record), toPutRequest(record).getItem());
+                    log.info("Sending request to DynamoDB API");
+                    PutItemResult putItemResult = client.putItem(tableName(record), toPutRequest(record).getItem());
+                    log.info(String.format("Request sent, %s WCU consumed", putItemResult.getConsumedCapacity()));
```

Comment:
```suggestion
                    log.info("Request sent, {} WCU consumed", putItemResult.getConsumedCapacity());
```
saves interpolating the string if the log if info's disabled (though sadly not the call to get the capacity)